test/utils.jl

```diff
@@ -80,34 +77,25 @@ module TestUtils
             3rd Quartile:   3.250000
             Maximum:        4.000000
             Length:         4
-            Type:           $Int
+            Type:           Int64
 
             nullarr
             Summary Stats:
-            Mean:           3.500000
-            Minimum:        2.000000
-            1st Quartile:   2.750000
-            Median:         3.500000
-            3rd Quartile:   4.250000
-            Maximum:        5.000000
             Length:         4
-            Type:           $Int
-            Number Missing: 0
-            % Missing:      0.000000
+            Type:           Union{Int64, Nulls.Null}
```

Comment:
If you checkout the full diff here, you'll see that previously, this column was detected as `Int` type and they have additional data points like mean, quantiles, etc. Now that it's changed to type `Union{Int, Null}`, we don't get all of that previous "numerical statistics" describe information, but it's treated rather as a string column or something.